### DISCUSSION TITLE
Account suspended

### DISCUSSION THREAD

**Discussion by rofaya** [community/community#145196] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Can I get help in accessing my suspended account because I don't know why I got suspended without any notice. I have submitted an email to GitHub Support, but I have not received a reply for a month. (2024/10/24)

（；´д｀）ゞ

id: QIN2DIM

**Comment by Ravindulakmina** [comment]
My other account has been suspended. **I got it successfully**. Below is how I got it.

Submit a request to GitHub support and reserve a ticket. Explain what happened to you there. Probably you have made a mistake. Get advice to resolve it and ask to get the account.
[github support](https://support.github.com/request)

If you do it correctly you will receive an email from github support or from the email below. It also has your ticket number. You also have the convenience of replying to it.

[developer@githubsupport.com](developer@githubsupport.com)

You can easily access the report below where it shows "suspended" while logging into the account.
Apply a correct reply to the email you receive.

I explained to you how I got my account, if you found it useful, **please name it as an answer**.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @rofaya,  

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
